go/tasks/pluginmachinery/flytek8s/config/config.go

```diff
@@ -167,6 +168,8 @@ type K8sPluginConfig struct {
 	// DefaultPodTemplateResync defines the frequency at which the k8s informer resyncs the default
 	// pod template resources.
 	DefaultPodTemplateResync config2.Duration `json:"default-pod-template-resync" pflag:",Frequency of resyncing default pod templates"`
+
+	SendObjectEvents bool `json:"send-object-events" pflag:",If true, will send object events as TaskExecutionEvent updates."`
```

Comment:
Can we add a `k8s` designation here somewhere? `SendK8sObjectEvents` maybe? or in the description. Would be nice to be explicit here.

Reply:
Added to the description since the struct itself already has k8s in the name, but happy to add to the field if you feel that's clearer.